source/include/core_mqtt_serializer.h

```diff
@@ -96,9 +96,11 @@ typedef enum MQTTStatus
     MQTTIllegalState,     /**< An illegal state in the state record. */
     MQTTStateCollision,   /**< A collision with an existing state record entry. */
     MQTTKeepAliveTimeout, /**< Timeout while waiting for PINGRESP. */
-    MQTTNeedMoreBytes     /**< MQTT_ProcessLoop/MQTT_ReceiveLoop has received
+    MQTTNeedMoreBytes,     /**< MQTT_ProcessLoop/MQTT_ReceiveLoop has received
                           incomplete data; it should be called again (probably after
                           a delay). */
+    MQTTAlreadyConnected, /**< MQTT Connection is established with the broker */
```

Comment:
MQTTConnected is an item of the enum MQTTConnectionStatus_t and this new name "MQTTAlreadyConnected" is supposed be an item of the enum MQTTStatus_t. So that is the reason we had to use a different name.